packages/integration-sdk-runtime/src/logger/index.ts

```diff
@@ -413,18 +413,18 @@ export function createErrorEventDescription(
   let errorCode: string;
   let errorReason: string;
 
-  if (err instanceof IntegrationError) {
```

Comment:
I don't understand what I'm looking at. You're saying that this is returning `false` for `IntegrationProviderAuthenticationError`, and thus using the default unexpected `errorCode` and `errorReason` properties?

This test suggests that the condition here should be returning true, so I'm not sure why this _wouldn't_ be setting the correct error message... 

```ts
test.only('auth', () => {
  const err = new IntegrationProviderAuthenticationError({
    endpoint: '',
    status: '', 
    statusText: '',
  });

  expect(err instanceof IntegrationError).toBe(true);
});
```

Can you try explaining it to me like I'm five? 🙃

Reply:
Whoops I got a few things conflated when tackling another bug. I think you're right. I'll close soon.